core/server/web/middleware/validation/profile-image.js

```diff
@@ -0,0 +1,23 @@
+var common = require('../../../lib/common'),
+    imageLib = require('../../../lib/image');
+
+module.exports = function profileImage() {
```

Comment:
If we don't have to configure the middleware, we don't need to return or use a closure here.